src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -0,0 +1,28 @@
+package seedu.bankwithus;
+
+public class BankWithUs {
+
+    private Storage storage;
+    private Ui ui;
+    private AccountList accounts;
+
+    public static final String FILE_PATH = "data/save.txt";
```

Comment:
Fails a CI build task with Error: Static variable definition in wrong order. [DeclarationOrder]
declare variables in order to resolve check style violation:
1. Constants (public static final)
2. Static variables (public static)
2. Instance variables (private)